services/web/server/src/simcore_service_webserver/resource_manager/module_setup.py

```diff
@@ -9,31 +9,43 @@
 import logging
 
 from aiohttp import web
-from servicelib.aiohttp.application_setup import ModuleCategory, app_module_setup
+from servicelib.aiohttp.application_setup import (
+    AlreadyInitializedError,
+    ModuleCategory,
+    app_module_setup,
+)
 
+from ..garbage_collector import setup_garbage_collector
+from ..redis import setup_redis
 from .config import APP_CLIENT_SOCKET_REGISTRY_KEY, APP_RESOURCE_MANAGER_TASKS_KEY
-from .garbage_collector import setup_garbage_collector
-from .redis import setup_redis_client
 from .registry import RedisResourceRegistry
-from .settings import assert_valid_config
 
 logger = logging.getLogger(__name__)
 
 
 @app_module_setup(
-    "simcore_service_webserver.resource_manager", ModuleCategory.SYSTEM, logger=logger
+    "simcore_service_webserver.resource_manager", ModuleCategory.ADDON, logger=logger
 )
 def setup_resource_manager(app: web.Application) -> bool:
     """Sets up resource manager subsystem in the application"""
 
     # ----------------------------------------------
     # TODO: temporary, just to check compatibility between
     # trafaret and pydantic schemas
-    assert_valid_config(app)
+    # assert_valid_config(app)
     # ---------------------------------------------
 
     app[APP_RESOURCE_MANAGER_TASKS_KEY] = []
-    setup_redis_client(app)
+    try:
+        setup_redis(app)
+    except AlreadyInitializedError as err:
+        logger.info("Skips setting up redis client: %s", err)
```

Comment:
Why when catching `AlreadyInitializedError` you print that it skips setting up the Redis client. Also should this not be in warning?

Reply:
also agree here: if it is already initialized, the message should go along the lines of ```redis client alreaddy initialized``` (which is what you already have inside the exception). Since redis might be setup at different places I guess this is a temporary?

Reply:
on second thought, i agree with your view. 
I changed the mechanism. The setup decorator will skip this call if performed more than once